### PR TITLE
Added the config option to snooze the notification of a continuous faulty key

### PR DESCRIPTION
**Pull request by iulianpascalau**
- added the config option to snooze the notification of a continuous faulty key

Solves the issue #26

### CODE REVIEW

File: cmd/monitor/config/example/config.toml [hunk 1]
```diff
@@ -10,6 +10,10 @@
     [General.Logs]
         LogFileLifeSpanInMB = 1024 # 1GB
         LogFileLifeSpanInSec = 86400 # 1 day
+    [General.AlarmSnooze]
+        Enabled = true
+        NumNotificationsForEachFaultyKey = 3
+        SnoozeTimeInSec = 86400 # 1 day
```
Comment (andreibancioiu) [inline cmd/monitor/config/example/config.toml:16]: Maybe a smaller value as a default? E.g. 8 / 12 hours?

File: executors/blsKeysTimeCache.go [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+	defer timeCache.mutFaultyBLSKeys.Unlock()
+
+	currentTimestamp := timeCache.getCurrentTimestamp()
+
+	info := timeCache.faultyBLSKeys[blsKey]
+	if info == nil {
+		info = &faultyBLSKeyInfo{
+			addedTimestamp: currentTimestamp,
+		}
+		timeCache.faultyBLSKeys[blsKey] = info
```
Comment (andreibancioiu) [inline executors/blsKeysTimeCache.go:51]: This in-memory storage is not cleaned / entries are not removed, but that is not a concern (as there are only a few items, at most thousands) :+1:

**Review by andreibancioiu — DISMISSED** [review]

File: executors/blsKeysTimeCache.go (hunk 2 above)
Comment (iulianpascalau) [reply executors/blsKeysTimeCache.go:51]: exactly. Didn't want it to complicate it with cleaning go routines. Can be added at a later time, if desired. :)

File: cmd/monitor/config/example/config.toml (hunk 1 above)
Comment (iulianpascalau) [reply cmd/monitor/config/example/config.toml:16]: done. Also added documentation

**Review by andreibancioiu — DISMISSED** [review]

**Review by andreibancioiu — APPROVED** [review]

**Review by miiu96 — APPROVED** [review]

**Pull request merged**